src/features/query-bar/__tests__/QueryBar-test.js

```diff
@@ -19,4 +19,15 @@ describe('features/query-bar/components/QueryBar', () => {
         });
         expect(wrapper).toMatchSnapshot();
     });
+
+    test('should render ColumnButton with columns that do not include item properties', () => {
+        const wrapper = getWrapper({
+            columns,
+        });
+        const FilterButton = wrapper.find('FilterButton');
+        expect(FilterButton.props().columns).toEqual(columnsWithoutItems);
+        const ColumnButton = wrapper.find('ColumnButton');
+        expect(ColumnButton.props().columns).toEqual(columnsWithoutName);
+        expect(wrapper).toMatchSnapshot();
```

Comment:
Let's omit this. It's an onus on reviewers but I'm not sure if it brings any obvious value.